benchmark/util/text-encoder.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
 const common = require('../common.js');
+const assert = require('assert');
 
 const bench = common.createBenchmark(main, {
-  len: [16, 32, 256, 1024, 1024 * 32],
-  n: [1e4],
+  len: [16, 32, 256, 1024, 1024 * 8],
```

Comment:
```suggestion
  len: [32, 256, 1024, 1024 * 8],
```

It is unlikely that 16 characters have a significant other performance profile to 32 characters. As such, limiting the variations seems a good possibility?